docs/build/windows.md

```diff
@@ -101,19 +101,24 @@ Building:
 * Ant
 * Gradle
 * Maven
+* OpenJDK 11 (for 17.0)
+* OpenJDK 8 (for stable-16.0)
```

Comment:
```suggestion
* OpenJDK 11
```

We don't need to maintain docs for older versions in the alpha branch -- you'd look at the `stable-16.0` branch to read docs for that version.